pypowsybl/network.py

```diff
@@ -500,6 +508,23 @@ def update_shunt_compensators(self, df: pd.DataFrame):
         """
         return self.update_elements(_pypowsybl.ElementType.SHUNT_COMPENSATOR, df)
 
+    def update_non_linear_shunt_comensator_sections(self, df):
```

Comment:
For now there is nothing to update and update does not work well for multi-indexed dataframes --> I will remove it before merging